src/modules/showList.js

```diff
@@ -0,0 +1,64 @@
+import { TaskList } from './classList.js';
+
+export const todo = new TaskList();
+const listSection = document.querySelector('.list-section');
+
+export const create = () => {
+  listSection.replaceChildren();
+  if (todo.todos.length > 0) {
+    listSection.style.display = 'block';
+    const listContainer = document.createElement('ul');
+    listContainer.className = 'allTodos';
+    listSection.appendChild(listContainer);
+    todo.todos.map((a) => {
+      const list = document.createElement('li');
+      list.className = 'todo';
+
+      const descriptionContainer = document.createElement('div');
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.id = 'checkbox';
+      if (a.completed === true) {
+        checkbox.checked = 'checked';
+      }
+
+      checkbox.onclick = (e) => {
+        todo.completed(e.target.checked, a.index);
+      };
+
```

Comment:
Your checkbox works great but there is no line through on the completed todos. Kindly add that and if anything is not clear reach out to me.

Reply:
Thank you for your review. I have worked on this issue and fixed it.
![Screenshot from 2022-02-22 01-07-01](https://user-images.githubusercontent.com/60083437/155014222-c15c368e-e559-4aa2-990d-7a4ab725fc8d.png)